Common Programming Concepts/Functions/Extract Method Refactoring/src/main.rs

```diff
@@ -0,0 +1,7 @@
+fn main() {
+    let year1993 = 1993;
+    println!("{}: 10 years ago was {}", year1993, year1993 - 10);
+
+    let year2020 = 2020;
```

Comment:
I understand, this is just an abstract example, and we probably will not change this over the years, but as we release it in 2021, maybe we should switch to year2021 = 2021?